alembic/versions/205ce02374b3_add_received_collateral_address_to_liquidations.py

```diff
@@ -0,0 +1,27 @@
+"""Add received_collateral_address to liquidations
+
+Revision ID: 205ce02374b3
+Revises: c8363617aa07
+Create Date: 2021-10-04 19:52:40.017084
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "205ce02374b3"
+down_revision = "c8363617aa07"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "liquidations",
+        sa.Column("received_token_address", sa.String(256), nullable=True),
+    )
+
+
+def downgrade():
+    op.drop_table("liquidations")
```

Comment:
Just noticed this - should be a remove column, not dropping the table

Always good to test both downgrade and upgrade by upgrading, downgrading, then upgrading again
```
alembic upgrade head
alembic downgrade -1
alembic upgrade head
```

Reply:
Gotcha